api/v1alpha1/envoygateway_types.go

```diff
@@ -503,11 +503,13 @@ type ExtensionManager struct {
 	// FailOpen defines if Envoy Gateway should ignore errors returned from the Extension Service hooks.
 	// The default is false, which means Envoy Gateway will fail closed if the Extension Service returns an error.
 	//
-	// Fail-close means that if the Extension Service hooks return an error, the relevant route/listener/resource
-	// will be replaced with a default configuration returning Internal Server Error (HTTP 500).
+	// Fail-close means that if the Extension Service hooks return an error, the error is propagated to the
+	// xds-translator. As a result, the xds configuration produced by the translator for that IR key is not
```

Comment:
nit: xds-translator and IR key may be internal implementation details that users are not familiar with. Or do we expect them to understand these details if they are running an extension server?

Reply:
+1, maybe we can say xDS tied to a specific Envoy Proxy fleet